### PR TITLE
Add known paths for more recent versions of msbuild

### PR DESCRIPTION
**Pull request by alcated**
### Description

Since Visual Studio 2017, two 2 major versions were released.
Here is the update to add the known paths for thoses two new versions.

## TODO

Feel free to open the PR and ask for help

- [ ] New (API-)documentation for new features exist (Note: API-docs are enough, additional docs are in `help/markdown`)
- [ ] unit or integration test exists (or short reasoning why it doesn't make sense)
  
  > Note: Consider using the `CreateProcess` API which can be tested more easily, see https://github.com/fsharp/FAKE/pull/2131/files#diff-4fb4a77e110fbbe8210205dfe022389b for an example (the changes in the `DotNet.Testing.NUnit` module)
  
- [ ] boy scout rule: "leave the code behind in a better state than you found it" (fix warnings, obsolete members or code-style in the places you worked in)     
- [ ] (if new module) the module has been linked from the "Modules" menu, edit `help/templates/template.cshtml`, linking to the API-reference is fine.
- [ ] (if new module) the module is in the correct namespace
- [ ] (if new module) the module is added to Fake.sln (`dotnet sln Fake.sln add src/app/Fake.*/Fake.*.fsproj`)
- [ ] Fake 5 [API guideline](https://fake.build/contributing.html#API-Design-Guidelines) is honored

### CODE REVIEW

File: .config/dotnet-tools.json [hunk 1]
```diff
@@ -15,10 +15,10 @@
       ]
     },
     "fake-cli": {
-      "version": "5.21.0",
+      "version": "1.0.0-1",
```
Comment (yazeedobaid) [inline .config/dotnet-tools.json:18]: Can you please revert this change also? It seems this is an automatic commit
we will try to fix it and remove it.

Comment (alcated) [reply .config/dotnet-tools.json]: I rebased onto release/next.
Is it acceptable to you ?

Comment (yazeedobaid) [reply .config/dotnet-tools.json]: yes thanks

**Comment by NogginBops** [comment]
This will actually not find the Visual Studio 2022 files as they are in "Program Files" but "Program Files (x86)" is the only folder searched.
